Tests/SmithyRetriesTests/DefaultRetryStrategy/DefaultRetryStrategyTests.swift

```diff
@@ -28,7 +28,7 @@ final class DefaultRetryStrategyTests: XCTestCase {
 
     override func setUp() {
         backoffStrategy = .init()
-        backoffStrategy.random = { 1.0 }
+        backoffStrategy.random = { @Sendable () -> Double in 1.0 }
```

Comment:
IDK why I have to explicitly mark these blocks as Sendable, this is a pretty easy case so the compiler must not do any auto-checking.